multicluster/controllers/multicluster/serviceexport_controller.go

```diff
@@ -257,20 +235,7 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 			Namespace: req.Namespace,
 		},
 	}
-
-	err = r.Client.Get(ctx, req.NamespacedName, ep)
-	if err != nil {
-		klog.ErrorS(err, "Failed to get Endpoints", "endpoints", req.String())
-		if apierrors.IsNotFound(err) && epInstalled {
-			err = r.handleEndpointDeleteEvent(ctx, req, commonArea)
-			if err != nil {
-				return ctrl.Result{}, err
-			}
-			r.installedEps.Delete(epObj)
-		}
-		return ctrl.Result{}, client.IgnoreNotFound(err)
-	}
-
+	ep.Subsets = []corev1.EndpointSubset{common.GetSubsets(svc)}
 	if epInstalled {
 		installedEp := epObj.(*epInfo)
 		if apiequality.Semantic.DeepEqual(getEndPointsPorts(ep), installedEp.ports) &&
```

Comment:
Should we change how to detect no EP change? As we just need to check if Service ClusterIP/port changed, we no long need to check individual Endpoints?

Reply:
done, removed Endpoint comparison.